kzz7yh-f108684/cod-ve09v2_kzz7yh-f108684.xml

```diff
@@ -503,7 +507,7 @@
     </p>
                <p xml:id="kzz7yh-f108684-d1e870">
                   <lb ed="#V" n="39"/>Ad primum in oppositum dicendum quod illud verbum 
-      <lb ed="#V" n="40"/>Aug. intelligendum est vel de 
+      <lb ed="#V" n="40"/><ref>Aug.</ref><!--Not sure where this reference ends--> intelligendum est vel de 
```

Comment:
I'm not sure i see this as a reference, but more like a mention. "that the word of augustine". Possible "word of augustine" could be a reference, perhaps an internal reference to something discussed above. But we'd need to find that.

For now I suggest. 

```suggestion
      <lb ed="#V" n="40"/><name>Aug.</name> intelligendum est vel de 
```